src/libraries/System.Private.CoreLib/src/System/Type.Helpers.cs

```diff
@@ -332,6 +332,9 @@ public virtual bool IsSubclassOf(Type c)
             return false;
         }
 
+        public bool IsAssignableTo(Type? targetType)
```

Comment:
Can add a nullability annotation like `IsAssignableFrom`?

```suggestion
        public bool IsAssignableTo([NotNullWhen(true)] Type? targetType)
```

Reply:
Done

Reply:
@benaadams I think you need to update the ref/ source after adding the attribute.

Reply:
Yeah :)